docs/content/stable/yugabyte-platform/manage-deployments/edit-helm-overrides.md

```diff
@@ -15,4 +15,26 @@ If your universe was created using Kubernetes, you have an option of modifying t
 
 ![img](/images/yb-platform/kubernetes-config66.png)
 
-Complete the dialog by following instructions provided in [Configure Helm overrides](../../create-deployments/create-universe-multi-zone-kubernetes#configure-helm-overrides).
\ No newline at end of file
+Complete the dialog by following instructions provided in [Configure Helm overrides](../../create-deployments/create-universe-multi-zone-kubernetes#configure-helm-overrides).
+
+## Upgrade universes for GKE Service Account based IAM support
+
+Existing universes can be software upgraded to versions which support [GKE Service Account based IAM](../../back-up-restore-universes/configure-backup-storage/#gke-service-account-based-iam-gcp-iam) using the following steps:
+
+1. Upgrade YugabyteDB to a version which supports the feature ( 2.18.x or later). For more details, refer to [Upgrade the YugabyteDB software](../../manage-deployments/upgrade-software/).
+
+1. Apply overrides using the "Edit Kubernetes overrides" flow.
+
+   **Prerequisite:** Provide the Kube namespace of the YugabyteDB Universe pods with the annotated Service Account. This may not be present already, as the service account would not have been used in YugabyteDB Universe pods.
```

Comment:
```suggestion
   **Prerequisite:** Provide the Kube namespace of the YugabyteDB universe pods with the annotated Kubernetes service account. This may not be present already, as the service account would not have been used in YugabyteDB Universe pods.
```